Terminal.Gui/Views/TextView.cs

```diff
@@ -1794,6 +1799,14 @@ void WrapTextModel ()
 		/// </summary>
 		public ContextMenu ContextMenu { get; private set; }
 
+		/// <summary>
+		/// Gets or sets if null terminated in a buffer array is considered as part of the <see cref="Text"/>.
+		/// </summary>
+		public bool AllowsNullTerminated {
```

Comment:
What is the user-scenario where a developer would set this property to new?

It seems like very obscure edge-case?

Reply:
See the unit test added. If it's encoded as `Unicode` it add a null terminated in the array byte.